### PR TITLE
Add Artifactory package.

### PR DESCRIPTION
**Pull request by ssk2**
cc @jainishshah17.

### CODE REVIEW

**Comment by rji** [comment]
@ssk2, @jainishshah17 — Just a heads-up that I just force pushed to this branch with all of the changes from here and Jainish's fork of the Universe. This should now be the latest version of the Artifactory package. There are still a few changes that need to be made, so let's not merge this one just yet.

File: repo/packages/A/artifactory/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "assets": {
+        "container": {
+            "docker": {
+                "artifactory-pro": "jainishshah17/artifactory:latest",
+                "artifactory-ha-primary": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-ha-primary:4.8.2",
+                "artifactory-ha-secondary": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-ha-secondary:4.8.2"
+            }
+        }
+    },
+    "cli":{
```
Comment (rji) [inline repo/packages/A/artifactory/0/resource.json]: This is a Universe 3.x feature, so we'll need to bump the `packagingVersion`. But I think this requires a DC/OS-specific subcommand CLI, not a generic CLI :/

**Comment by jainishshah17** [comment]
@rji : I have added changes we discussed Yesterday in [fork](https://github.com/Shahbrothers/universe/tree/artifactory-universe-3x)

**Comment by rji** [comment]
@jainishshah17 — updated PR based on your fork

**Comment by jainishshah17** [comment]
@ssk2 @rji : can you please change Artifactory-Pro image from ```jainishshah17/artifactory:latest``` to ```jfrog-docker-reg2.bintray.io/jfrog/dcos-artifactory-pro:4.8.2``` in resource.json

**Comment by rji** [comment]
@jainishshah17 Updated!

File: repo/packages/A/artifactory/0/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "packagingVersion": "3.0",
+    "name": "artifactory",
+    "version": "4.8.2",
+    "minDcosReleaseVersion" : "1.8",
+    "scm": "https://github.com/JfrogDev/artifactory-mesos.git",
+    "maintainer": "soleng-team@jfrog.com",
+    "website": "https://jfrog.com",
+    "framework": true,
```
Comment (ssk2) [inline repo/packages/A/artifactory/0/package.json]: This should be false.

**Comment by ssk2** [comment]
@rji, @jainishshah17: I just force pushed a rebased version of this branch.

**Comment by ssk2** [comment]
@BenWhitehead: let me know when you've managed to update the whitelist. Then we're ready to go :)

**Pull request merged**